CMakeLists.txt

```diff
@@ -292,6 +292,10 @@ if (NOT USE_SIMD STREQUAL "")
             else ()
                 add_definitions ("-m${feature}")
             endif ()
+            if (feature STREQUAL "fma" AND CMAKE_COMPILER_IS_CLANG)
+                # Clang needs an explicit flag to make fma ops
+                add_definitions ("-ffp-contract=fast")
```

Comment:
But you either opt-in for the entire code-base, or nothing at all.

There is no way to get the benefit of FMA only for the fast math functions for example.